applications/CompressiblePotentialFlowApplication/python_scripts/compute_lift_process.py

```diff
@@ -14,46 +26,90 @@ def __init__(self, Model, settings ):
             "model_part_name": "please specify the model part that contains the surface nodes",
             "velocity_infinity": [1.0,0.0,0.0],
             "reference_area": 1.0,
+            "moment_reference_point" : [],
             "create_output_file": false
         }''')
 
         settings.ValidateAndAssignDefaults(default_parameters)
 
         self.body_model_part = Model[settings["model_part_name"].GetString()]
-        self.velocity_infinity = [0,0,0]
-        self.velocity_infinity[0] = settings["velocity_infinity"][0].GetDouble()
-        self.velocity_infinity[1] = settings["velocity_infinity"][1].GetDouble()
-        self.velocity_infinity[2] = settings["velocity_infinity"][2].GetDouble()
+        self.fluid_model_part = self.body_model_part.GetRootModelPart()
         self.reference_area =  settings["reference_area"].GetDouble()
         self.create_output_file = settings["create_output_file"].GetBool()
+        self.moment_reference_point = settings["moment_reference_point"].GetVector()
+        
 
     def ExecuteFinalizeSolutionStep(self):
         KratosMultiphysics.Logger.PrintInfo('ComputeLiftProcess','COMPUTE LIFT')
 
-        rx = 0.0
-        ry = 0.0
-        rz = 0.0
+        total_force = KratosMultiphysics.Vector(3)
+        total_moment = KratosMultiphysics.Vector(3)
 
         for cond in self.body_model_part.Conditions:
             n = cond.GetValue(KratosMultiphysics.NORMAL)
             cp = cond.GetValue(KratosMultiphysics.PRESSURE)
 
-            rx += n[0]*cp
-            ry += n[1]*cp
-            rz += n[2]*cp
+            # Computing forces
+            total_force += n.__mul__(cp)
```

Comment:
do you need the `__mul__`?

Reply:
This now works. I used to get the following error:

total_force = n*cp
TypeError: unsupported operand type(s) for *: 'Kratos.Vector' and 'float'